.travis.yml

```diff
@@ -103,6 +103,8 @@ before_install:
       sed -i.bak '/^coala/d' requirements.txt
       sed -i.bak '/^mypy-lang/d' requirements.txt bear-requirements.txt
     fi
+  - sudo pear install PHP_CodeSniffer
+  - phpcbf --help
```

Comment:
do we need this?

Reply:
no actually i was just testing(actually I had ubuntu:16 so there just installing php-codesniffer puts `phpcbf` in path).